asm/interpreter.asm

```diff
@@ -146,26 +146,16 @@ interpret_tib
     jmp PUTCHR
 +   rts
 
-.on_stack_underflow
-    lda #$12 ; reverse on
-    jsr PUTCHR
-    lda #'e'
-    jsr PUTCHR
-    lda #'r'
-    jsr PUTCHR
-    jmp .stop_error_print
-
-.on_data_underflow
-    lda #$12 ; reverse on
-    jsr PUTCHR
-    lda #'f'
-    jsr PUTCHR
-    lda #'u'
-    jsr PUTCHR
-    lda #'l'
-    jsr PUTCHR
-    lda #$d
-    jmp PUTCHR
+.throw_stack_underflow
+    ldy #-4
+    jmp throw_y
+.throw_dictionary_overflow
+    ldy #-8
+    ; fall through
+throw_y
+    lda #$ff
+    jsr pushya
```

Comment:
Am I mistaken or are your Y's and A's mixed up? I thought `pushya` put Y in MSB and A in LSB.

Reply:
no wonder it didn't work! nice catch!